cmd/cli/cmd.go

```diff
@@ -335,6 +340,9 @@ func NewCommand() (*cobra.Command, *int) {
 	runCommand.PersistentFlags().Bool("dns-check", config.ViperConfig.GetBool("dns-check"), "needed dns queries to notify readiness")
 	config.ViperConfig.BindPFlag("dns-check", runCommand.PersistentFlags().Lookup("dns-check"))
 
+	runCommand.PersistentFlags().Bool("skip-probe", config.ViperConfig.GetBool("skip-probe"), "skip probing systemd units and kubelet healthz")
+	config.ViperConfig.BindPFlag("skip-probe", runCommand.PersistentFlags().Lookup("skip-probe"))
+
```

Comment:
Should this be plural...`skip-probes`? Does it ever make sense to skip systemd unit probes but not kubelet healthz probe?

Maybe `skip-etcd-probe` `skip-kubelet-probe`?